src/basic/cgroup-util.c

```diff
@@ -254,22 +254,6 @@ bool cg_ns_supported(void) {
         return (supported = false);
 }
 
-bool cg_freezer_supported(void) {
-        static thread_local int supported = -1;
-
-        if (supported >= 0)
-                return supported;
-
-        if (cg_all_unified() <= 0)
-                return (supported = false);
-
-        if (access("/sys/fs/cgroup/init.scope/cgroup.freeze", F_OK) >= 0)
```

Comment:
As already noted in README, cgroup.freeze is supported since kernel v5.2 (https://github.com/torvalds/linux/commit/76f969e8948d82e78e1bc4beb6b9465908e74873). Hence, OK.